examples/getstarted/src/admin/app.example.js

```diff
@@ -0,0 +1,9 @@
+const config = {
```

Comment:
@Convly I have remove the custom file from the getstarted app and I have just put the new format here .

The new format will help us to provide typings for the StrapiAdmin class.